README.md

```diff
@@ -127,64 +224,63 @@ final class CustomProvider: FeatureProvider {
         // resolve a boolean flag value
     }
 
-    func getStringEvaluation(
-        key: String,
-        defaultValue: String,
-        context: EvaluationContext?
-    ) throws -> ProviderEvaluation<String> {
-        // resolve a string flag value
+    ...
+}
+
+```
+> Built a new provider? [Let us know](https://github.com/open-feature/openfeature.dev/issues/new?assignees=&labels=provider&projects=&template=document-provider.yaml&title=%5BProvider%5D%3A+) so we can add it to the docs!
+
+### Develop a hook
+
+To develop a hook, you need to create a new project and include the OpenFeature SDK as a dependency.
+This can be a new repository or included in [the existing contrib repository](https://github.com/open-feature/swift-sdk-contrib) available under the OpenFeature organization.
```

Comment:
note: Contrib repo has not been created yet

Reply:
Same comment as above.